tests/migrations/test_autodetector.py

```diff
@@ -2785,6 +2786,7 @@ def test_add_field_and_unique_together(self):
             unique_together={("title", "newfield")},
         )
 
+    @ignore_warnings(category=RemovedInDjango51Warning)
```

Comment:
This shouldn't be necessary anymore. Please decorate the entire `AutodetectorIndexTogetherTests` class.

Reply:
Good catch 👍